cognee/cli/commands/tui_command.py

```
@@ -0,0 +1,54 @@
+            action="store_true",
+            help="Disable mouse support (keyboard only mode)",
+        )
+    
+    def execute(self, args: argparse.Namespace) -> None:
+        try:
+            fmt.echo("Starting Cognee TUI...")
+            fmt.note("Press 'q' to quit, '?' for help")
+            
+            # Import and run TUI
+            from cognee.cli.tui import run_tui
+            run_tui(mouse=not args.no_mouse)
```

Pylint (4.0.2)

[error] Unexpected keyword argument 'mouse' in function call

(E1123)